src/uraft/saunafs-uraft-helper.in

```diff
@@ -151,13 +151,47 @@ saunafs_metadata_version() {
 	esac
 }
 
+have_master_role() {
+	local local_status="$(saunafs_admin metadataserver-status --porcelain "${matocl_host}" "${matocl_port}")"
+	[ "$(awk '{print $1}' <<< "${local_status}" 2>/dev/null)" == "master" ]
+}
+
+is_ip_present() {
+	local ip="${1}"
+	local iface="${2:-}"
+	local -a ip_command_extra_args=()
+	if [ -n "${iface}" ]; then
+		ip_command_extra_args+=("dev" "${iface}")
+	fi
+	ip addr show "${ip_command_extra_args[@]}" | grep -wq -m1 "${ip}"
+}
+
+is_floating_ip_present() {
+	if is_ip_present "${ipaddr}" "${iface}"; then
+		return 0
+	fi
+	if [ -n "${ipaddr2}" ] && is_ip_present "${ipaddr2}" "${iface2}"; then
```

Comment:
Maybe the question is silly :sweat_smile:, but I have a doubt: 
Assuming `${ipaddr2}` is different than `${ipaddr}`, does the cluster will recognize `${ipaddr2}` as the usual floating IP and will work correctly as in the scenario where floating IP will be `${ipaddr}`?